tests/test_annubes_env.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+import pytest
+
+from neurogym.envs.annubes import AnnubesEnv
+
+RND_SEED = 42
+
+
+@pytest.fixture
+def default_env():
+    """Fixture for creating a default AnnubesEnv instance."""
+    return AnnubesEnv()
+
+
+@pytest.fixture
+def custom_env():
+    """Fixture for creating a custom AnnubesEnv instance with specific parameters."""
+    return AnnubesEnv(
+        session={"v": 1},
+        stim_intensities=[0.5, 1.0],
+        stim_time=800,
+        catch_prob=0.3,
+        fix_intensity=0.1,
+        fix_time=300,
+        dt=50,
+        tau=80,
+        n_outputs=3,
+        output_behavior=[0, 0.5, 1],
```

Comment:
I think `n_outputs` needs to match `len(output_behavior)`, right? In that case, should we get rid of the parameter?
I will create a PR for this.